youki_integration_test/src/tests/lifecycle/container_lifecycle.rs

```diff
@@ -1,20 +1,33 @@
-use std::path::{Path, PathBuf};
-
-use crate::support::generate_uuid;
+use crate::utils::{generate_uuid, prepare_bundle, TempDir};
+use std::thread::sleep;
+use std::time::Duration;
 use test_framework::{TestResult, TestableGroup};
 
 use super::{create, delete, kill, start, state};
 
+// By experimenting, somewhere around 50 is enough for youki process
+// to get the kill signal and shut down
+// here we add a little buffer time as well
+const SLEEP_TIME: u64 = 75;
```

Comment:
How about using `std::time::Duration` insted ot `u64`.

Reply:
A duration is created later from this u64, and then given to sleep because I thought we could not invoke functions for defining consts, but I checked ans we can use `duration::from_millis` directly in constant definition. I will change that along with other required changes in additional commit :+1: